src/Yasumi/Provider/CzechRepublic.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/**
+ *  This file is part of the Yasumi package.
+ *
+ *  Copyright (c) 2015 - 2016 AzuyaLabs
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ *
+ * @author Sacha Telgenhof <stelgenhof@gmail.com>
+ */
+
+namespace Yasumi\Provider;
+
+use DateTime;
+use DateTimeZone;
+use Yasumi\Holiday;
+
+/**
+ * Provider for all holidays in Czech republic.
+ *
+ * Class CzechRepublic
+ * @package Yasumi\Provider
+ * @author Dennis Fridrich <fridrich.dennis@gmail.com>
```

Comment:
I believe you have copied the Germany Test classes :) No problem,but maybe good to remove/adjust all references to the other class and authors.